### PR TITLE
Bitesize fixes

### PR DESCRIPTION
**Pull request by AndyMoore111**
- bring the change over from st2contrib (https://github.com/StackStorm/st2contrib/pull/637#issuecomment-266611429)
- fix region connection for cloudformation

### CODE REVIEW

**Comment by ErikThoreson** [comment]
+1

File: actions/lib/action.py [hunk 1]
```diff
@@ -53,6 +53,11 @@ def vpc_connect(self):
         del self.credentials['region']
         return boto.vpc.connect_to_region(region, **self.credentials)
 
+    def cf_connect(self):
+        region = self.credentials['region']
+        del self.credentials['region']
+        return boto.cloudformation.connect_to_region(region, **self.credentials)
```
[the remark this answers is not in the document]
Comment (AndyMoore111) [reply actions/lib/action.py:59]: yes it seems to be good

[the remark this answers is not in the document]
Comment (userlocalhost) [reply actions/lib/action.py:59]: Thank you for your reply.
I see. It seems that is imported internally.

But I think it's a good habit to import module that you use expressly for removing internal dependency.
And I recommend you to write the test for showing it's all right, when possible.

The document how to do it is prepared.
https://docs.stackstorm.com/development/pack_testing.html

Thank you.

**Comment by userlocalhost** [comment]
@AndyMoore111

The points that you mentioned in this PR are independent.
> * bring the change over from st2contrib (StackStorm/st2contrib#637 (comment))
> * fix region connection for cloudformation

The former correction looks good to me.

If you decouple the later correction to another PR, it's easy for maintainers to merge this patch, I think.

**Comment by userlocalhost** [comment]
+1
because I add the test that covers this correction at #8

**Comment by emedvedev** [comment]
Thanks a lot for the contribution! Took me some time to get to it: vacation and the long holidays. :) Merging.

**Pull request merged**